java-frontend/src/main/java/org/sonar/java/se/checks/UnclosedResourcesCheck.java

```diff
@@ -19,7 +19,15 @@
  */
 package org.sonar.java.se.checks;
 
+import java.util.ArrayList;
```

Comment:
Please discard these changes, and only limit the modifications to the new imports.